### PR TITLE
Added implementation for QEAuto-21 C5930

### PR DESCRIPTION
**Pull request by wMercadoJala**

### CODE REVIEW

File: IDT.MVNO/Test/UI/MVNO/Feature/Savings Calculator.feature [hunk 1]
```diff
@@ -0,0 +1,68 @@
+﻿Feature: Savings Calculator
+	In order to Test Saving Calculators
+	As a common user
+	I want to estimate the use of data with mvno and compare with a provider's bill 
+
+Background: 
+	Given I login to "MVNO Staging" portal web with "MVNO Staging" credentials
+
+@mvno @savingcalculator
+Scenario: C36670: Verify Minutes Used increased for Saving Calculator
```
Comment (mabalderramav) [inline IDT.MVNO/Test/UI/MVNO/Feature/Savings Calculator.feature]: I think, is better Are these scenarios convert to a scenario outline?
What do you think?

File: IDT.MVNO/Main/UI/Pages/Home/HomePage.cs [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+            CommonActions.WaitUntilWebElemenHasAttribute(outputElement, "class", "hidden");
+            CommonActions.SendKeys(inputElement, valueInput);
+        }
+
+        /// <summary>
+        /// Fills a input in Calculator based on type of field.
+        /// </summary>
+        /// <param name="valueInput">Value of the input.</param>
+        /// <param name="field">Type of Field, it could be Minutes, SMS or Data.</param>
+        public void FillTxtField(string valueInput, string fieldType)
```
[the remark this answers is not in the document]
Comment (wMercadoJala) [reply IDT.MVNO/Main/UI/Pages/Home/HomePage.cs]: done

File: IDT.MVNO/Main/UI/Pages/Home/HomePage.cs [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+            actions.MoveToElement(this.LblTotalAmount).Perform();
+        }
+
+        /// <summary>
+        /// Fills the inputs in Calculator section.
+        /// </summary>
+        /// <param name="outputElement">WebElement to focus on input.</param>
+        /// <param name="inputElement">WebElement to fill with a value.</param>
+        /// <param name="valueInput">Value for the input.</param>
+        public static void FillInputCalculator(IWebElement outputElement, IWebElement inputElement, string valueInput)
```
[the remark this answers is not in the document]
Comment (wMercadoJala) [reply IDT.MVNO/Main/UI/Pages/Home/HomePage.cs]: fixed

File: IDT.MVNO/Test/UI/MVNO/Feature/Savings Calculator.feature (hunk 1 above)
Comment (wMercadoJala) [reply IDT.MVNO/Test/UI/MVNO/Feature/Savings Calculator.feature]: We need to research about how to update test cases in Test Rail for Scenarios Outline

Comment (mabalderramav) [reply IDT.MVNO/Test/UI/MVNO/Feature/Savings Calculator.feature]: Ok

**Pull request merged**